migrate.py

```diff
@@ -565,118 +482,118 @@ def mark_moved_resources(checkout_dir, collection, migrated_to_collection):
-            depth = depth.lstrip('/')
-            depth = os.path.dirname(depth)
-            depth = depth.split('/')
-            rel_path = '.'.join(['' for x in range(-1, len(depth))])
-
-            with open(unit_file, 'r') as f:
-                unit_lines = f.readlines()
-            unit_lines = [x.rstrip() for x in unit_lines]
-
-            changed = False
-
-            for module in module_names:
-                for li,line in enumerate(unit_lines):
-                    if line.startswith('from ') and line.endswith(module):
-                        unit_lines[li] = 'from %s.plugins.modules import %s' % (rel_path, module)
-                        changed = True
-
-            if changed:
-                with open(unit_file, 'w') as f:
-                    f.write('\n'.join(unit_lines))
-            #import epdb; epdb.st()
-
-
-        list_of_targets = []  # TODO: same as above require from spec or find for ourselves?
-        if list_of_targets:
-            dst = os.path.join(cdir, 'test', 'integration', 'targets')
-            if not os.path.exists(dst):
-                os.makedirs(dst)
-            for uf in v['targets']:
-                fuf = os.path.join(args.vardir, 'test', 'integration', 'targets', uf)
-                duf = os.path.join(dst, os.path.basename(fuf))
-                if not os.path.exists(os.path.join(dst, os.path.basename(fuf))):
-                    try:
-                        shutil.copytree(fuf, duf)
-                    except Exception as e:
-                        import epdb; epdb.st()
-
-                # set namespace for all module refs
-                cmd = 'find %s -type f -name "*.yml"' % (duf)
-                res = run_command(cmd)
-                yfiles = res['so'].split('\n')
-                yfiles = [x.strip() for x in yfiles if x.strip()]
-
-                for yf in yfiles:
-                    with open(yf, 'r') as f:
-                        ydata = f.read()
-                    _ydata = ydata[:]
-
-                    for module in v['modules']:
-                        msrc = os.path.basename(module)
-                        msrc = msrc.replace('.py', '')
-                        msrc = msrc.replace('.ps1', '')
-                        msrc = msrc.replace('.ps2', '')
-
-                        mdst = '%s.%s.%s' % (args.namespace, coll, msrc)
-
-                        if msrc not in ydata or mdst in ydata:
-                            continue
-
-                        #import epdb; epdb.st()
-                        ydata = ydata.replace(msrc, mdst)
-
-                    # fix import_role calls?
-                    #tasks = yaml.load(ydata)
-                    #import epdb; epdb.st()
-
-                    if ydata != _ydata:
-                        logger.info('fixing module calls in %s' % yf)
-                        with open(yf, 'w') as f:
-                            f.write(ydata)
+#def copy_tests(plugin, coll, spec, args):
```

Comment:
I think since it's not called, there's no need to comment this out.

Reply:
MIght be a guide for when we write the functionality. Once we rewrite it, we can remove it.

Reply:
Yes. Just don't comment it out. It's more useful in its normal state as editors will highlight it.

Reply:
thanks